release/tag/cloudbuild.yaml

```diff
@@ -187,18 +98,10 @@ steps:
 
   # push the container image
   - name: 'gcr.io/cloud-builders/docker'
-    args: ['push', 'gcr.io/kpt-dev/example-functions:${TAG_NAME}']
+    args: ['push', 'gcr.io/$PROJECT_ID/example-functions:${TAG_NAME}']
 
   # update latest tag on release
   - name: 'gcr.io/cloud-builders/docker'
-    args: ['push', 'gcr.io/kpt-dev/example-functions:latest']
-
-  # push the binaries
-  - name: 'gcr.io/cloud-builders/gsutil'
-    args: ['cp', '-r', '-a', 'public-read', '/workspace/releases/', 'gs://kpt-dev/']
```

Comment:
Are we still copying `kpt` to the `gs://kpt-dev` gsutil bucket?

Reply:
That is a good point. Currently it does not upload to the gs bucket. Users should migrate over to download from the github releases, but we should probably give it some time.